### PR TITLE
Jobs CLI

### PR DESCRIPTION
**Pull request by andrewmchen**

### CODE REVIEW

File: databricks_cli/jobs/api.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from databricks_cli.configure.config import get_jobs_client
+
+
+def create_job(json):
+    return get_jobs_client().client.perform_query('POST', '/jobs/create', data=json)
```
Comment (loneknightpy) [inline databricks_cli/jobs/api.py]: why do we need to call perform_query directly?

File: databricks_cli/jobs/cli.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+@require_config
+@eat_exceptions
+def get_cli(job_id):
+    click.echo(pretty_format(get_job(job_id)))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--job-id', required=True, type=int)
+@click.option('--jar-params', default=None)
+@click.option('--notebook-params', default=None)
```
Comment (loneknightpy) [inline databricks_cli/jobs/cli.py]: how about python jobs and spark submit job?

File: databricks_cli/jobs/api.py (hunk 1 above)
Comment (andrewmchen) [reply databricks_cli/jobs/api.py]: The jobs client doesn't have a nice way to invoke an API given the entire JSON. You always have to unpack the top level fields unfortunately.

File: databricks_cli/jobs/cli.py (hunk 2 above)
Comment (andrewmchen) [reply databricks_cli/jobs/cli.py]: Let's add them once we document these fields in the API.

**Pull request merged**